src/main/scala/org/broadinstitute/hail/driver/ExportVCF.scala

```diff
@@ -50,6 +50,28 @@ object ExportVCF extends Command {
     case _ => "String"
   }
 
+  def appendIntArrayOption(sb: StringBuilder, toAppend: Option[Array[Int]]): Unit = {
+    if (toAppend.isDefined)
+      toAppend.foreach(_.foreachBetween(sb.append(_))(sb += ','))
+    else
+      sb += '.'
+  }
+
+  def appendIntOption(sb: StringBuilder, toAppend: Option[Int]): Unit = {
+    if (toAppend.isDefined)
+      sb.append(toAppend)
```

Comment:
This is appending the `Option`, not its contents.  This was causing some tests to fail.  You want `toAppend.get`, but better is pattern matching:

```
  toAppend match {
    case Some(i) => sb.append(i)
    case None => sb += '.'
  }
```

Same for `appendIntArrayOption `.  Otherwise, looks great!